app/test_main.py

```diff
@@ -1 +1,49 @@
-# write your code here
+from unittest import mock
+from unittest.mock import Mock
+
+import pytest
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "valid_value,connection_value,result",
+    [
+        pytest.param(
+            True,
+            True,
+            "Accessible",
+            id="can access google page if 2 value true"
+        ),
+        pytest.param(
+            True,
+            False,
+            "Not accessible",
+            id="can access google page if valid value true"
+        ),
+        pytest.param(
+            False,
+            False,
+            "Not accessible",
+            id="can access google page if 2 value false"
+        ),
+        pytest.param(
+            False,
+            True,
+            "Not accessible",
+            id="can access google page if connection value true"
+        )
+    ]
+)
+@mock.patch("app.main.has_internet_connection")
+@mock.patch("app.main.valid_google_url")
+def test_can_access_google_page_if_2_value_true(
```

Comment:
The test function name `test_can_access_google_page_if_2_value_true` might be misleading as it suggests the test only covers the scenario where both values are true. However, the parameterized test cases cover multiple scenarios. Consider renaming the function to something more descriptive of the overall test, such as `test_can_access_google_page_various_conditions`.